webapp/graphite/metrics/views.py

```diff
@@ -290,6 +295,19 @@ def tree_json(nodes, base_path, wildcards=False):
   return results
 
 
+def nodes_by_position(matches, position):
+  found = set()
+
+  for metric in matches:
+    nodes = metric.path.split('.')
+    node = nodes[position]
+    if node in found:
+      continue
+    found.add(node)
+    results = "{'nodes': %s}" % sorted(list(found))
```

Comment:
Doesn't this line have an extra level of indentation?

`sorted(found)` is enough, a set is already an iterable.

Is the format meant to be JSON? Or something else? Serialization shouldn't be done with string interpolation.